setup/scaling/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: single
+title:  "Scale Spinnaker"
+sidebar:
+  nav: setup
+---
+
+Spinnaker is a large system, made of multiple (up to 10) microservices, each
```

Comment:
Currently 11, right? Should we just say "made up of _many_ microservices"?

Reply:
I thought 10, but you're right, "many" read better